modules/tm/cluster.c

```diff
@@ -473,9 +479,6 @@ int tm_reply_replicate(struct sip_msg *msg)
 	if (!tm_cluster_enabled())
 		return 0;
 
-	/* double-check we have received the message on a anycast network */
```

Comment:
This is literally checking again once we've already verified it's either bypassed or the check itself has passed, this is a pointless piece of code in my eyes so I'm just ripping it out

Reply:
Ignore this, I'm going to put it back in and do the anycast check